.containerignore

```diff
@@ -0,0 +1,9 @@
+# This file is derived of the .dockerignore
+# It is required to pass `.git` to the builder containers on RHTAP so we can bake build information into the binary.
+# It is required to pass the `.gitignore` files in `image/bin/` and `bin` to avoid a dirty version.
+# RHTAP uses Buildah which first looks up for `.containerignore`, then, if absent, `.dockerignore`.
+# Having this file enables us to have different exclusion list for RHTAP builds than in GHA/OSCI.
+# To regenerate, replace the below contents with the output of the following command (please keep these comments).
+#  grep -vF -e '/.git/' -e '/bin/' -e '/image/bin/' .dockerignore
+
+# currently empty!
```

Comment:
Could you add that it is empty and must be kept to prevent RHTAP from using `.dockerignore` file?

Reply:
Thanks, I followed your suggestion.